docs/UserGuide.md

```diff
@@ -116,105 +118,124 @@ Note that each command might use the prefixes slightly differently so refer to e
 
 1. Amounts can be exactly `$0.00` as users may want to simply track that a transaction is present but not specify the amount.
+User might also want to track financial events not involving currency exchange, such as barter trading, free gifts, etc. 
 2. Intervals work by filtering by the specified time period. 
    1. For `day` intervals, only transactions of the same day are found. 
    2. For `week` intervals, only transactions of the same [week of year](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/temporal/WeekFields.html#weekOfYear()) are found. 
    3. For `month` intervals, only transactions of the same month are found.
 
-### UI Layout
+[//]: # ()
 
-UniCa$h is designed with users who prefer to use the keyboard in mind. Thus, almost all
-user input is designed for CLI-type usage, i.e. text-based keyboard input, and UI elements are intended 
-to supplement this main functionality.
+[//]: # (UI layout and description of what each section means)
 
-When UniCa$h is first opened, it would look something like this:
+### User Interface (UI) Layout
 
-![img_1.png](images/unicash/UniCashWelcome.png)
+UniCa$h is designed with users who prefer to use the keyboard in mind. Thus, almost all
+user input is designed for CLI-type usage, i.e. text-based keyboard input, and User Interface
+elements are intended to either supplement this main functionality, or provide graphical support
+for features.
+
+When UniCa$h is first opened, it would look something like the image shown earlier 
+near the top of the user guide.
 
-By default, the Welcome Message will be displayed in the `Results Display`.
+By default, the UniCa$h welcome message will be displayed in the `Results Display`.
 This message can also be invoked with the `help` command which will be explained
 later on in this User Guide. Below are the main User Interface (UI) component features
 we have implemented in UniCa$h.
 
 ![img_2.png](images/unicash/UniCashUIAnnotated.png)
 
-Explained below are the main UI components. For the purposes of demonstrating certain UI features, certain commands
-and inputs that are yet to be explained are mentioned here. However, at a later section of this User Guide,
-all of these commands and inputs will be explained, feel free to refer to them at your discretion. _Where applicable,
-consider those explanations as the single source of authority for those commands as the representation here is merely
-for UI demonstration purposes only._
 
+These main UI components are explained below. For the purposes of demonstrating certain UI features, certain commands
+and inputs that are yet to be explained are mentioned here. However, at a later section of this User Guide,
+all of these commands and inputs will be explained, feel free to refer to them at your discretion. **Where applicable,
+consider those explanations as the single source of authority for those commands, as the representation here is merely
+for UI demonstration purposes only.**
```

Comment:
Feels like this section is not needed for the readers, maybe keep the last line in a callout but the top seems self-explanatory

Reply:
Yup, I've moved this to the DG as well